config/routes/listing.rb

```diff
@@ -7,12 +7,18 @@
 end
 
 namespace :api, defaults: { format: "json" } do
-  scope module: :v0, constraints: ApiConstraints.new(version: 0, default: true) do
+  scope module: :v1, constraints: ApiConstraints.new(version: 1, default: false) do
     resources :listings, only: %i[index show create update]
     get "/listings/category/:category", to: "listings#index", as: :listings_category
     get "/organizations/:organization_username/listings", to: "organizations#listings",
                                                           as: :organization_listings
   end
+
+  scope module: :v0, constraints: ApiConstraints.new(version: 0, default: true) do
+    resources :listings, only: %i[index show create update]
+    get "/listings/category/:category", to: "listings#index"
+    get "/organizations/:organization_username/listings", to: "organizations#listings"
+  end
```

Comment:
While testing this PR I realized the v1 routes for these weren't included until now